Arashayo/TabletopHaven
# Modify group backend

## Related Issue
https://app.clickup.com/t/86bxjny22

## Changes list
Combined 2 api endpoints into 1 (since they are used in only 1, the same view)
Made group name display in group details

## Testing instructions
Go to group view, check if everything works as expected, check if group name displays.

## Screenshots (if appropriate):
